src/Common/src/Interop/Unix/libssl/Interop.libssl.msbuild

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
```

Comment:
This msbuild idea was from @vijaykota. A lot of crypto/ssl related files that I am including here are also included in System.Net.Security project and a couple of others.
And at least till now, there was a lot of churn around the names of files and  their locations. 
Idea was to introduce this file and import it from all the projects needed, so that it is easier to change.
Please let me know if this is not the right way or if there's a better way.